audiomate/corpus/io/common_voice.py

```diff
@@ -86,72 +132,64 @@ def _load(self, path):
 
         return corpus
 
+    # ==============================================================================================
+
     @staticmethod
     def get_subset_ids(path):
-        """ Return a list with ids of all available subsets (based on existing csv-files). """
-        subset_ids = []
+        """ Return a list with ids of all available subsets (based on existing tsv-files) """
 
-        for subset_path in glob.glob(os.path.join(path, '*.tsv')):
+        subset_ids = []
+        for subset_path in glob.glob(os.path.join(path, "*.tsv")):
             file_name = os.path.split(subset_path)[1]
             basename = os.path.splitext(file_name)[0]
 
-            # We don't want to include the invalidated files
-            # since there maybe corrupt files
-            if basename != 'invalidated':
+            # We don't want to include the invalidated files since there maybe corrupt files
+            if basename != "invalidated" and basename != "reported":
                 subset_ids.append(basename)
 
         return subset_ids
 
+    # ==============================================================================================
+
     def load_subset(self, corpus, path, subset_idx):
-        """ Load subset into corpus. """
-        csv_file = os.path.join(path, '{}.tsv'.format(subset_idx))
-        subset_utt_ids = []
-
-        entries = textfile.read_separated_lines_generator(
-            csv_file,
-            separator='\t',
-            max_columns=8,
-            ignore_lines_starting_with=['client_id'],
-            keep_empty=True
-        )
+        """ Load subset into corpus """
 
-        for entry in entries:
-            file_idx = self.create_assets_if_needed(
-                corpus,
-                path,
-                entry
-            )
+        tsv_file = os.path.join(path, "{}.tsv".format(subset_idx))
 
-            if file_idx is not None:
-                subset_utt_ids.append(file_idx)
+        data = pd.read_csv(tsv_file, keep_default_na=False, sep="\t")
```

Comment:
I had some problems with the _textfile_ reader, sometimes data rows were shorter than others an I got IndexNotFound errors (I'm assuming multiple "\t" were not splitted correctly). Also the CV update changed column order so it's more resistant against future changes if we can use named access.
I could have fixed the _textfile_ reader but it was much easier to use pandas, which is also widely known and I assuming it can also improve performance because optimized for this kind of tasks.

Reply:
We already have too many dependencies and pandas is a big one. I prefer that we fix our textfile reader. Otherwise we should replace our own code across the board with pandas. But again, this cannot be part of this PR.